package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "cuttle",
   "description": "The deepest card game under the sea",
-  "version": "4.3.29",
+  "version": "4.3.30",
```

I'm going to merge one other update first so please pull from `main` and bump this to 4.3.31
```suggestion
  "version": "4.3.31",
```